### PR TITLE
October newsletter

### PR DESCRIPTION
**Pull request by meenaxc**

### CODE REVIEW

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+<a href="pitchlarger90degrees.png">
+<img src="pitchlarger90degrees.png" width="100%"/>
+</a>
+<br/>
+<br/>
+
+An interesting read on how this works can be found [here] (https://github.com/maplibre/maplibre-gl-js/blob/49d7fc96ad883b4e1042fe305cd0c0a16ccd9732/developer-guides/center-point.md)
+
+WWe have introduced some breaking changes in these versions as was planned for version 5, to make sure to take a look at the changelog for more details.
```
Comment (HarelM) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:35]:
```suggestion
We have introduced some breaking changes in these versions as was planned for version 5, so make sure to take a look at the changelog for more details.
```

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<a href="pitchlarger90degrees.png">
+<img src="pitchlarger90degrees.png" width="100%"/>
+</a>
+<br/>
+<br/>
+
+An interesting read on how this works can be found [here] (https://github.com/maplibre/maplibre-gl-js/blob/49d7fc96ad883b4e1042fe305cd0c0a16ccd9732/developer-guides/center-point.md)
+
+We have introduced some breaking changes in these versions as was planned for version 5, so make sure to take a look at the changelog for more details.
+There are still open issues with globe and other improvement we plan to add as part of this major release.
```
Comment (HarelM) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:36]:
```suggestion
There are still open issues with globe and other improvements we plan to add as part of this major release. Stay tuned!
```

File: content/news/2024-10-30-maplibre-newsltter-october-2024/index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
```
Comment (HarelM) [inline content/news/2024-10-30-maplibre-newsltter-october-2024/index.md:1]: This looks like a duplication...?

File: content/roadmap/leightweight-renderers/image.jpg
Comment (HarelM) [inline content/roadmap/leightweight-renderers/image.jpg:1]: How is this file related to this PR?

Comment (ramyaragupathy) [reply content/roadmap/leightweight-renderers/image.jpg:1]: @meenaxc - I think you have to branch out from the latest, because this change was made available on main branch two weeks ago through https://github.com/maplibre/maplibre.github.io/pull/346

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- We also made two releases for [MapLibre Android v10](https://github.com/maplibre/maplibre-native/releases?q=android-v10&expanded=true) with some backports.
+
+## Flutter
+
+We’re conducting a brief survey to gather feedback on MapLibre Flutter. The survey takes only about 2 minutes to complete and will help to identify strengths and areas for improvement. If you've worked with MapLibre on Flutter, your input is highly appreciated.
+[You can participate here](https://forms.gle/UTB6KjVFtmYv2RzU8)
+
+## Conferences
+
+Check out these events to attend:
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:63]: This is not formatted properly.

You can run Hugo locally with:

```
npm install
npm run serve
```

Screenshot 2024-11-05 at 20 50 49

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+- MapLibre iOS [6.7.1 and 6.8.0] were released. (https://github.com/maplibre/maplibre-native/releases?q=ios&expanded=true).
+
+- MapLibre Android [11.5.1] was released. (https://github.com/maplibre/maplibre-native/releases?q=android-v11&expanded=true).
+
+- We also made two releases for [MapLibre Android v10](https://github.com/maplibre/maplibre-native/releases?q=android-v10&expanded=true) with some backports.
+
+## Flutter
+
+We’re conducting a brief survey to gather feedback on MapLibre Flutter. The survey takes only about 2 minutes to complete and will help to identify strengths and areas for improvement. If you've worked with MapLibre on Flutter, your input is highly appreciated.
+[You can participate here](https://forms.gle/UTB6KjVFtmYv2RzU8)
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:59]: You need to make here a m
```suggestion
You can participate [here](https://forms.gle/UTB6KjVFtmYv2RzU8).
```

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+Here's an interesting situation that should be possible now:
+
+<a href="pitchlarger90degrees.png">
+<img src="pitchlarger90degrees.png" width="100%"/>
+</a>
+<br/>
+<br/>
+
+An interesting read on how this works can be found [here] (https://github.com/maplibre/maplibre-gl-js/blob/49d7fc96ad883b4e1042fe305cd0c0a16ccd9732/developer-guides/center-point.md)
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:32]: This Markdown link is broken.

See Markdown syntax here: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+We have introduced some breaking changes in these versions as was planned for version 5, so make sure to take a look at the changelog for more details.
+There are still open issues with globe and other improvements we plan to add as part of this major release. Stay tuned!
+See the initial post in the following issue to see the current status:
+
+[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834] (https://github.com/maplibre/maplibre-gl-js/issues/3834)
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:38]: Broken Markdown link.

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+## MapLibre Native
+
+- Development of the Vulkan backend continued this month. We also ran another benchmark on AWS Device Farm, using even more device types this time. One conclusion you may draw from the results is that Vulkan will offer a solid performance boost on modern devices. We're still validating these results. Stay tuned for an Android official release following later this year!
+
+- Kaushal Kumar Singh rewrote the Android build config from Groovy to the more modern Kotlin in [#2902](https://github.com/maplibre/maplibre-native/pull/2902).
+
+-Support for text-variable-anchor-offset has been merged [#2921](hhttps://github.com/maplibre/maplibre-native/pull/2921). This feature is one of the features that Microsoft championed to be added to MapLibre GL JS, and now ported to MapLibre Native. We're happy with this contribution and are excited about Microsoft adopting MapLibre Native.
+
+- MapLibre iOS [6.7.1 and 6.8.0] were released. (https://github.com/maplibre/maplibre-native/releases?q=ios&expanded=true).
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:50]: Broken Markdown link.

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+- MapLibre Android [11.5.1] was released. (https://github.com/maplibre/maplibre-native/releases?q=android-v11&expanded=true).
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:52]: Broken Markdown link.

**Review by louwers — CHANGES_REQUESTED** [review]
I have formatted the Markdown files with

```
npm run format
```

I also deleted some files that I think were committed by accident.

I left some comments.

**Comment by meenaxc** [comment]
Have made edits, please check!

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 10]
```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
...
 
-[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834] (https://github.com/maplibre/maplibre-gl-js/issues/3834)
+[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834](https://github.com/maplibre/maplibre-gl-js/issues/3834)
 
 Overall we are progressing well and ironing out the globe feature.
 
 ## MapLibre Native
 
-- Development of the Vulkan backend continued this month. We also ran another benchmark on AWS Device Farm, using even more device types this time. One conclusion you may draw from the results is that Vulkan will offer a solid performance boost on modern devices. We're still validating these results. Stay tuned for an Android official release following later this year!
+For MapLibre Native, we have continued development of Vulkan, along with Android and IoS updates and feature parity.
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:44]: IoS -> wrong capitalization.

I don't think this sentence adds anything.

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 11]
```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
...
+[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834](https://github.com/maplibre/maplibre-gl-js/issues/3834)
 
 Overall we are progressing well and ironing out the globe feature.
 
 ## MapLibre Native
 
-- Development of the Vulkan backend continued this month. We also ran another benchmark on AWS Device Farm, using even more device types this time. One conclusion you may draw from the results is that Vulkan will offer a solid performance boost on modern devices. We're still validating these results. Stay tuned for an Android official release following later this year!
+For MapLibre Native, we have continued development of Vulkan, along with Android and IoS updates and feature parity. 
 
+- Development of the Vulkan backend continued this month. We also ran another benchmark on AWS Device Farm, using even more device types this time. One conclusion you may draw from the results is that Vulkan will offer a solid performance boost on modern devices. We're still validating these results. Stay tuned for an Android official release following later this year!
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:46]: Link to the results is missing?

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 12]
```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
...
 - Kaushal Kumar Singh rewrote the Android build config from Groovy to the more modern Kotlin in [#2902](https://github.com/maplibre/maplibre-native/pull/2902).
-
 -Support for text-variable-anchor-offset has been merged [#2921](hhttps://github.com/maplibre/maplibre-native/pull/2921). This feature is one of the features that Microsoft championed to be added to MapLibre GL JS, and now ported to MapLibre Native. We're happy with this contribution and are excited about Microsoft adopting MapLibre Native.
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:48]: Please copy the Markdown text I submitted.

There is an error in the link now `hhtps` instead of `https`. Also the link to `text-variable-anchor-offset` is gone as are the backticks.

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 13]
```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
...
 
-An interesting read on how this works can be found [here] (https://github.com/maplibre/maplibre-gl-js/blob/49d7fc96ad883b4e1042fe305cd0c0a16ccd9732/developer-guides/center-point.md)
+An interesting read on how this works can be found [here](https://github.com/maplibre/maplibre-gl-js/blob/49d7fc96ad883b4e1042fe305cd0c0a16ccd9732/developer-guides/center-point.md)
 
 We have introduced some breaking changes in these versions as was planned for version 5, so make sure to take a look at the changelog for more details.
 There are still open issues with globe and other improvements we plan to add as part of this major release. Stay tuned!
 See the initial post in the following issue to see the current status:
 
-[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834] (https://github.com/maplibre/maplibre-gl-js/issues/3834)
+[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834](https://github.com/maplibre/maplibre-gl-js/issues/3834)
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:38]: It would be clearer to make `initial post` the link here and replace the `:` with a `.` in the sentence...

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 14]
```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
 <br/>
 <br/>
 
-An interesting read on how this works can be found [here] (https://github.com/maplibre/maplibre-gl-js/blob/49d7fc96ad883b4e1042fe305cd0c0a16ccd9732/developer-guides/center-point.md)
+An interesting read on how this works can be found [here](https://github.com/maplibre/maplibre-gl-js/blob/49d7fc96ad883b4e1042fe305cd0c0a16ccd9732/developer-guides/center-point.md)
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:32]: Missing dot at the end of the sentence.

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 15]
```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
...
+- MapLibre iOS [6.7.1 and 6.8.0](https://github.com/maplibre/maplibre-native/releases?q=ios&expanded=true) were released.
 - MapLibre Android [11.5.1] was released. (https://github.com/maplibre/maplibre-native/releases?q=android-v11&expanded=true).
-
 - We also made two releases for [MapLibre Android v10](https://github.com/maplibre/maplibre-native/releases?q=android-v10&expanded=true) with some backports.
 
 ## Flutter
 
 We’re conducting a brief survey to gather feedback on MapLibre Flutter. The survey takes only about 2 minutes to complete and will help to identify strengths and areas for improvement. If you've worked with MapLibre on Flutter, your input is highly appreciated.
-[You can participate here](https://forms.gle/UTB6KjVFtmYv2RzU8)
+You can participate [here](https://forms.gle/UTB6KjVFtmYv2RzU8)
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:56]: Missing dot at the end of the sentence.

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 16]
```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
...
 
 ## Conferences
 
 Check out these events to attend:
-[FOSS4G2025Belem] (https://2024.foss4g.org/en/)
-[FOSSGIS2025] (https://fossgis-konferenz.de/2025/programm/)
+
+FOSS4G is hosted by the Open Source Geospatial Foundation (OSGeo), in collaboration with the local community.The foundation's goal is to encourage the use and collaborative development of community-led projects, data development and education.
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:62]: This feels a little bit out of place.

But if you want to include it, move this text to the FOSS4G point below, because FOSSGIS is a different conference. We should also add a text for FOSSGIS for consistency then.

Also there is a space missing here: `community.The`

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 17]
```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
...
+
+- [FOSS4G 2024 in Belem, Brazil](https://2024.foss4g.org/en/)
+- [FOSSGIS 2025 in Muenster, Germany](https://fossgis-konferenz.de/2025/programm/)
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:65]: Münster is the correct spelling. We should mention that this conference is in German.

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md [hunk 18]
```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
...
+An interesting read on how this works can be found [here](https://github.com/maplibre/maplibre-gl-js/blob/49d7fc96ad883b4e1042fe305cd0c0a16ccd9732/developer-guides/center-point.md)
 
 We have introduced some breaking changes in these versions as was planned for version 5, so make sure to take a look at the changelog for more details.
 There are still open issues with globe and other improvements we plan to add as part of this major release. Stay tuned!
 See the initial post in the following issue to see the current status:
 
-[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834] (https://github.com/maplibre/maplibre-gl-js/issues/3834)
+[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834](https://github.com/maplibre/maplibre-gl-js/issues/3834)
 
 Overall we are progressing well and ironing out the globe feature.
```
Comment (louwers) [inline content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:40]: I think this is duplicated, because we also say that "There are still open issues with globe and other improvements we plan to add as part of this major release. Stay tuned!" in the previous paragraph.

**Review by louwers — CHANGES_REQUESTED** [review]
There are still some issues.

When copying Markdown from a GitHub comment, make sure to press 'Edit' first.

image

A few mistakes were likely introduced by just copying the text from the rendered HTML.

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md (hunk 10 above)
Comment (meenaxc) [reply content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:44]: Keeping to have a segue into the section

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md (hunk 16 above)
Comment (meenaxc) [reply content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:62]: Removed descriptors, not necessary to market their conference in our newsletter

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md (hunk 15 above)
Comment (meenaxc) [reply content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:56]: Added after "here" otherwise I see this grammatically fine

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md (hunk 13 above)
Comment (meenaxc) [reply content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:38]: Up to GLJS, made the change if no further comment

File: content/news/2024-10-30-maplibre-newsletter-october-2024/index.md (hunk 18 above)
Comment (meenaxc) [reply content/news/2024-10-30-maplibre-newsletter-october-2024/index.md:40]: Up to GLJS, made the change if no further comment

**Comment by meenaxc** [comment]
Pushed changes, please check

**Review by louwers — CHANGES_REQUESTED** [review]
Some of the Markdown links are broken again.

Markdown is also sensitive to formatting. For example, this:

```
- Kaushal Kumar Singh rewrote the Android build config from Groovy to the more modern Kotlin in [#2902](https://github.com/maplibre/maplibre-native/pull/2902).
-Support for..
```

Results in:

image

Instead of there being a separate bullet point.

The link to `text-variable-anchor-offset` in the MapLibre Style Spec that was in my submission is also still not there.

I'll push a commit to fix these, but please take a look at the syntax of Markdown: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Review by louwers — APPROVED** [review]
